pyproject.toml

```diff
@@ -36,7 +36,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-pypika-tortoise = "^0.2.1"
+pypika-tortoise = { git = "https://github.com/waketzheng/pypika-tortoise.git", branch = "fix-enums-not-quoted" }
```

Comment:
We need to release a new pypika version before we can merge it. @abondar, can you please do it or let me know how we can do it!

Reply:
0.2.2 seems to be released already
https://github.com/tortoise/pypika-tortoise/releases/tag/v0.2.2